pkg/kubectl/deployment.go

```diff
@@ -27,24 +27,42 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-// DeploymentBasicGeneratorV1 supports stable generation of a deployment
-type DeploymentBasicGeneratorV1 struct {
+// BaseDeploymentGenerator: implement the common functionality of
```

Comment:
I chose this pattern because I'm used to writing docstrings in the imperative mood (like commit messages) and `golint` approves it. I like it a lot, but I'm willing to change it. Thoughts?